css-basics_social-card/css/styles.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+  margin: 10px;
+}
+body {
+  background-color: whitesmoke;
+}
+.card-wrapper {
+  width: 800px;
+  background-color: white;
+  border-radius: 12px;
+  box-shadow: 12px;
+}
+img {
+  border-radius: 12px;
+}
```

Comment:
you could also add some styling to "h2"(f.e. text-transform, font-size) and "p" (f.e. font-size, margin)